src/components/tabs/tabs.component.js

```diff
@@ -248,10 +247,10 @@ const Tabs = ({
     return (
       <TabsHeader
         align={align}
-        position={sidebarContext ? "left" : position}
+        position={isInSidebar ? "left" : position}
         role="tablist"
         extendedLine={extendedLine}
-        alternateStyling={variant === "alternate" || !!sidebarContext}
+        alternateStyling={variant === "alternate" || !!isInSidebar}
```

Comment:
```suggestion
        alternateStyling={variant === "alternate" || isInSidebar}
```
It's already a boolean now